packages/nodejs/src/config.ts

```diff
@@ -127,6 +127,10 @@ export class Configuration {
       ignoreActions: [],
       ignoreErrors: [],
       ignoreNamespaces: [],
+      instrumentHttp: true,
+      instrumentHttps: true,
```

Comment:
```suggestion
```

Since the HTTP and HTTPS settings have been merged into `instrumentHttp`, the `instrumentHttps` entry can be removed from this map.